src/client/sandbox/event/focus-blur.js

```diff
@@ -333,12 +350,30 @@ export default class FocusBlurSandbox extends SandboxBase {
         if (activeElement !== el)
             withoutHandlers = true;
 
+        // NOTE: we should prevent the native focus event, which appears immediately
+        // after el.blur() call and raise this event via event simulator.
```

Comment:
something like: `prevent native focus event that appears before the blur event, because it should be raised after blur`